CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ IMPROVEMENTS:
 BUG FIXES:
 
  * csi: Fixed a panic in the API affecting both plugins and volumes. [[GH-8655](https://github.com/hashicorp/nomad/issues/8655)]
+ * licensing (Enterprise): Fixed a bug where enterprise license was not loaded from raft after a snapshot was restored. [[GH-8737](https://github.com/hashicorp/nomad/issues/8737)]
```

Comment:
I would put it as `core` as it affects all agents and functionality pretty much - and emphasize user impact, e.g. "Fixed a bug where servers may self-terminate as licenses are ignored after a Raft snapshot restore"

Reply:
perfect thank you